web/crux-ui/locales/en/container.json

```diff
@@ -118,12 +118,21 @@
     "loadBalancer": "Load balancer",
     "useLoadBalancer": "Use load balancer",
     "extraLBAnnotations": "Extra LB annotations",
-    "placeholders": { "headerName": "Header name", "cpuUsageExample": "10m", "memoryUsageExample": "512Mi" },
     "labels": "Labels",
     "annotations": "Annotations",
     "service": "Service",
     "deployment": "Deployment",
-    "ingress": "Ingress"
+    "ingress": "Ingress",
+    "metrics": "Metrics",
+    "metricsPath": "Metrics path",
+    "metricsPort": "Metrics port",
+    "placeholders": {
+      "headerName": "Header name",
+      "cpuUsageExample": "10m",
+      "memoryUsageExample": "512Mi",
+      "metricsPath": "/metrics",
+      "metricsPort": "80"
```

Comment:
This is incorrect in the sense that it is not the default value, crane will attempt to use the first exposed port available if nothing explicit was given here.
Is this communicated any way? If not, could it be?

Reply:
Okay I'll rework the port selection and the placeholder.